packages/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.2
+
+* Added Video Capability with `pickVideo`. 
+* Updated Example for Preview of Video Captured
```

Comment:
Please use full verb phrases using standard US spelling and capitalization. E.g.
```md
* Added support for picking videos.
* Updated example app to show video preview.
```